### DISCUSSION TITLE
Bot to post new discussions to #suggestions

### DISCUSSION THREAD

**Discussion by BeefSteaker** [chromiecraft/chromiecraft#3319] [Suggestions]
I assume its possible to set up the GitHub bot to post new discussion links into the #suggestion channel like how the #changlog-chromie & #forum channel work. 

IMO this is a much better place to have suggestions, but it also seriously decreases visibility. Automatically posting new discussions to the discord in a non interacable channel would remedy this.

**Comment by 55Honey** [comment]
Hi @Rubadubscrub
If you wish to receive notifications for discussions/updates, GitHub allows to set personalised notifications already.

**Reply by BeefSteaker** [reply]
Yeah but that requires people to come on here first. You lose the community interaction by making it only be here

My suggestion would be the best of both the old and new system

**Comment by BeefSteaker** [comment]
Looking into it, I don't think you even need a bot, can be done with built in webhooks

https://gist.github.com/jagrosh/5b1761213e33fc5b54ec7f6379034a22

**Reply by Nyeriah** [reply]
The issue is that you can't filter a specific category I presume. So you end up flooded by the rest of the discussions as well, not just suggestions.
